migrate/3.0-restore.sh

```diff
@@ -72,12 +79,16 @@ init_options() {
             help_init_options
             exit 0
         ;;
-        -*|--*)     # unknown option
-            POSITIONAL+=("${1}") # save it in an array for later
+        -*)     # unknown option
+            if [ -n "$1" ] ;
+            then
+                POSITIONAL+=("${1}") # save it in an array for later
+            fi
             shift
         ;;
         *)          # namespace
-            NAMESPACE=${1}
+            export NAMESPACE
+            NAMESPACE=$(echo "${1}" | xargs)
```

Comment:
Maybe?
```suggestion
            export NAMESPACE=$(echo "${1}" | xargs)
```

Reply:
Nope, shellcheck didn't like that. Tried it first!